client/my-sites/checkout/composite-checkout/components/wp-checkout.js

```diff
@@ -277,26 +278,46 @@ export default function WPCheckout( {
 		[ onEvent ]
 	);
 
+	const onSummaryError = useCallback(
+		( error ) =>
+			onEvent( {
+				type: 'STEP_LOAD_ERROR',
+				payload: {
+					message: error,
+					stepId: 'summary',
+				},
+			} ),
+		[ onEvent ]
+	);
+
 	return (
 		<Checkout>
 			<CheckoutSummaryArea className={ isSummaryVisible ? 'is-visible' : '' }>
-				<CheckoutSummaryTitleLink onClick={ () => setIsSummaryVisible( ! isSummaryVisible ) }>
-					<CheckoutSummaryTitle>
-						<CheckoutSummaryTitleIcon icon="info-outline" size={ 20 } />
-						{ translate( 'Purchase Details' ) }
-						<CheckoutSummaryTitleToggle icon="keyboard_arrow_down" />
-					</CheckoutSummaryTitle>
-					<CheckoutSummaryTitlePrice className="wp-checkout__total-price">
-						{ total.amount.displayValue }
-					</CheckoutSummaryTitlePrice>
-				</CheckoutSummaryTitleLink>
-				<CheckoutSummaryBody>
-					<WPCheckoutOrderSummary
-						onChangePlanLength={ changePlanLength }
-						nextDomainIsFree={ responseCart?.next_domain_is_free }
-					/>
-					<SecondaryCartPromotions responseCart={ responseCart } addItemToCart={ addItemToCart } />
-				</CheckoutSummaryBody>
+				<CheckoutErrorBoundary
+					errorMessage={ translate( 'Sorry, there was an error loading this information.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'Sorry, there was an error loading this page.' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/error-boundary-checkout-summary-step)